### ISSUE TITLE
Sec 6.5.2 > Download URL - add URI/URL to the example?

### ISSUE THREAD

**Issue by jyucsiro** [opengeospatial/netcdf-ld#71]
The current example has this:
```
dcat:distribution [
 	a dcat:Distribution;
 	dcat:downloadURL <{}>;
 	dcat:mediaType [
 		a dct:MediaType;
 		dct:identifier "application/x-netcdf"
 	];
```

Should we add an example downloadURL? Or is this meant to signal something else? @adamml

**Comment by adamml** [comment]
@marqh will look at adding a NetCDF file so we can use a _real_ URL here

**Comment by marqh** [comment]
closed by #88
